conftest.py

```diff
@@ -19,9 +19,11 @@ def pytest_addoption(parser):
 
 def set_fuser(fuser):
     if fuser == "te":
-        # use default fuser (TE) and
-        # profiling executor settings
-        pass
+        torch._C._jit_set_profiling_executor(True)
+        torch._C._jit_set_profiling_mode(True)
+        torch._C._jit_override_can_fuse_on_cpu(False)
```

Comment:
this can probably be changed to True to enable nnc on cpu (torch.jit.fuser behavior changed in https://github.com/pytorch/pytorch/pull/74078)

Reply:
Updated the code. I copied the config from timm: https://github.com/rwightman/pytorch-image-models/blob/2f2b22d8c7889174dbf11b92c2d72d8587f9164b/timm/utils/jit.py#L28, so probably good to update there too.